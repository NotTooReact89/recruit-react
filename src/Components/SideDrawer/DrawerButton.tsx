import * as React from 'react';
import { IClickProps } from '..';
import './DrawerButton.css';

const DrawerButton = (props: IClickProps) => (
    <button className='toggle-button' onClick={props.click}>
        <div className='toggle-button__line' />
        <div className='toggle-button__line' />
        <div className='toggle-button__line' />
    </button>
);

export default DrawerButton;