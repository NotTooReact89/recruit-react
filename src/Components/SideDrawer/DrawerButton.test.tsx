import * as React from 'react';
import DrawerButton from './DrawerButton';
const renderer = require('react-test-renderer');

it('should render drawer button correctly', () => {
  const tree = renderer.create((
    <DrawerButton click={jest.fn()} />
  )).toJSON()
  expect(tree).toMatchSnapshot()
})