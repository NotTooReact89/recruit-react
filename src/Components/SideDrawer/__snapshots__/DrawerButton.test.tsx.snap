// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render drawer button correctly 1`] = `
<button
  className="toggle-button"
  onClick={[MockFunction]}
>
  <div
    className="toggle-button__line"
  />
  <div
    className="toggle-button__line"
  />
  <div
    className="toggle-button__line"
  />
</button>
`;
